### ISSUE TITLE
[BUG] Incorrect warning "WARNING: No fixes with time integration, atoms won't move"

### ISSUE THREAD

**Issue by bendjensen** [lammps/lammps#3083]
**Summary**

LAMMPS reports this warning despite the existence of time-integrating fixes and normal movement of atoms in a simulation.

**LAMMPS Version and Platform**

LAMMPS 27 Oct 2021, OSX
```
Large-scale Atomic/Molecular Massively Parallel Simulator - 27 Oct 2021
Git info (master / patch_27Oct2021-54-g515ef7bece)

Usage example: lmp_20211103 -var t 300 -echo screen -in in.alloy

List of command line options supported by this LAMMPS executable:

-echo none/screen/log/both  : echoing of input script (-e)
-help                       : print this help message (-h)
-in none/filename           : read input from file or stdin (default) (-i)
-kokkos on/off ...          : turn KOKKOS mode on or off (-k)
-log none/filename          : where to send log output (-l)
-mdi '<mdi flags>'          : pass flags to the MolSSI Driver Interface
-mpicolor color             : which exe in a multi-exe mpirun cmd (-m)
-cite                       : select citation reminder style (-c)
-nocite                     : disable citation reminder (-nc)
-package style ...          : invoke package command (-pk)
-partition size1 size2 ...  : assign partition sizes (-p)
-plog basename              : basename for partition logs (-pl)
-pscreen basename           : basename for partition screens (-ps)
-restart2data rfile dfile ... : convert restart to data file (-r2data)
-restart2dump rfile dgroup dstyle dfile ... 
                            : convert restart to dump file (-r2dump)
-reorder topology-specs     : processor reordering (-r)
-screen none/filename       : where to send screen output (-sc)
-skiprun                    : skip loops in run and minimize (-sr)
-suffix gpu/intel/opt/omp   : style suffix to apply (-sf)
-var varname value          : set index style variable (-v)

OS: Darwin 20.6.0 x86_64

Compiler: Clang C++ Apple LLVM 13.0.0 (clang-1300.0.29.3) with OpenMP not enabled
C++ standard: C++11
MPI v1.0: LAMMPS MPI STUBS for LAMMPS version 27 Oct 2021
...
```

**Expected Behavior**

Simulations with time integration do not produce the warning: "WARNING: No fixes with time integration, atoms won't move"

**Actual Behavior**

Simulations with time integration DO produce the warning: "WARNING: No fixes with time integration, atoms won't move"

**Steps to Reproduce**

The example `examples/tersoff/in.hBN_shift` produces this warning. (see log below)

**Further Information, Files, and Links**

```
diff log.lammps log.13Jan21.hBN_shift.g++.1 
1c1,3
< LAMMPS (27 Oct 2021)
---
> LAMMPS (24 Dec 2020)
> OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:94)
>   using 1 OpenMP thread(s) per MPI task
34,36c36,38
< timestep        0.001
< velocity        all create 300.0 4928459 loop geom
< fix             thermostat all nve
---
> timestep  	0.001
> velocity  	all create 300.0 4928459 loop geom
> fix       	thermostat all nve
45,46c47
< run             1000
< WARNING: No fixes with time integration, atoms won't move (src/lammps/lammps/src/verlet.cpp:60)
---
> run 		1000
72c73
< Loop time of 0.886516 on 1 procs for 1000 steps with 880 atoms
---
> Loop time of 0.998048 on 1 procs for 1000 steps with 880 atoms
74,75c75,76
< Performance: 97.460 ns/day, 0.246 hours/ns, 1128.011 timesteps/s
< 99.1% CPU use with 1 MPI tasks x no OpenMP threads
---
> Performance: 86.569 ns/day, 0.277 hours/ns, 1001.956 timesteps/s
> 98.3% CPU use with 1 MPI tasks x 1 OpenMP threads
80c81
< Pair    | 0.8719     | 0.8719     | 0.8719     |   0.0 | 98.35
---
> Pair    | 0.98001    | 0.98001    | 0.98001    |   0.0 | 98.19
82,85c83,86
< Comm    | 0.0054234  | 0.0054234  | 0.0054234  |   0.0 |  0.61
< Output  | 0.0002215  | 0.0002215  | 0.0002215  |   0.0 |  0.02
< Modify  | 0.0062868  | 0.0062868  | 0.0062868  |   0.0 |  0.71
< Other   |            | 0.002685   |            |       |  0.30
---
> Comm    | 0.006538   | 0.006538   | 0.006538   |   0.0 |  0.66
> Output  | 0.000224   | 0.000224   | 0.000224   |   0.0 |  0.02
> Modify  | 0.00781    | 0.00781    | 0.00781    |   0.0 |  0.78
> Other   |            | 0.003467   |            |       |  0.35
100c101
< Total wall time: 0:00:00
---
> Total wall time: 0:00:01
```

**Comment by akohlmey** [comment]
I cannot reproduce it with the 7 January 2022 version:
```
Large-scale Atomic/Molecular Massively Parallel Simulator - 7 Jan 2022
Git info (develop / patch_7Jan2022)

Usage example: /home/akohlmey/compile/lammps/build-test/lmp -var t 300 -echo screen -in in.alloy

[...]

OS: Linux "Fedora Linux 35 (Thirty Five)" 5.15.12-200.fc35.x86_64 x86_64

Compiler: GNU C++ 11.2.1 20211203 (Red Hat 11.2.1-7) with OpenMP 4.5
C++ standard: C++14
MPI v3.1: MPICH Version:	3.4.1
MPICH Release date:	Fri Jan 22 14:17:48 CST 2021
MPICH ABI:	13:10:1
```

**Comment by akohlmey** [comment]
After some quick search it is confirmed that this was a temporary issue that was resolved with commit: 49258e93017b3f009dbe3625fcaa67b1c3ed6224
